src/components/AudioDetail.vue

```diff
@@ -307,6 +328,104 @@ export default defineComponent({
       () => store.state.nowPlayingContinuously
     );
 
+    const pronunciationByPhrase = computed(() => {
+      let textArray: Array<string> = [];
+      accentPhrases.value?.forEach((accentPhrase) => {
+        let textString = "";
+        accentPhrase.moras.forEach((mora) => {
+          textString += mora.text;
+        });
+        if (accentPhrase.pauseMora) {
+          textString += "、";
+        }
+        textArray.push(textString);
+      });
+      return textArray;
+    });
+
+    const pitchHistory = new Map<string, number>();
```

Comment:
これは提案ですが、`pitchHistory`を用いず、常に隣接モーラから`pitch`を計算するのはどうでしょうか。
`UNDO/REDO`プルリクエストもあるので、ユーザーはそこまで困らないと思います。
https://github.com/Hiroshiba/voicevox/pull/138

今の実装だと、`phraseIndex`と`moraIndex`が偶然同じだった別の`pitchHistory`があると挙動が変わります。

また、Vuex(Flux)の思想には状態を一箇所に集めるという仕組みがあります。
`pitchHistory`はこのコンポーネントが持ち続けるイレギュラーな状態となり、コードが少しだけ乱雑になります。

---

This is a suggestion, how about always calculating the `pitch` from the adjacent mora instead of using `pitchHistory`?
There is also the `UNDO/REDO` pull request, so users wouldn't have that much trouble.

With the current implementation, the behavior changes if another `pitchHistory` exists where the `phraseIndex` and `moraIndex` are coincidentally the same.

Also, Vuex (Flux) has a concept of collecting states into one place.
`pitchHistory` will be an irregular state that this component keeps, and the code will be a bit cluttered.